src/views/List.jsx

```diff
@@ -1,9 +1,15 @@
-import { useState } from 'react';
-
+import { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
 import { ListItem } from '../components';
 
-export function List({ data }) {
+export function List({ data, listToken }) {
+	const navigate = useNavigate();
 	const [searchTerm, setSearchTerm] = useState('');
+	const [copied, setCopied] = useState(false);
+
+	useEffect(() => {
+		if (copied) setTimeout(() => setCopied(false), 2000);
+	}, [copied]);
```

Comment:
This is well written, easy to understand code, but I'm curious about why it's needed. Seems like it solves an issue that I'm unaware of!

Reply:
This is to display "Copied" dialog on the copy token box for 2 seconds after clicking it, then to "pop" it back to showing the list token value.

Reply:
Wow, it's so well done I just assumed it was something built-in/native! Nice touch!